src/main/java/com/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -1,4 +1,69 @@
 package com.novi.TechItEasy.controllers;
 
+import com.novi.TechItEasy.exceptions.RecordNotFoundException;
+import com.novi.TechItEasy.model.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/techItEasy")
 public class TelevisionController {
+    private List<Television> tv;// 1. declare the list
+
+    public TelevisionController() {
+        tv = new ArrayList<>();// 2. initialise the list
+        Television tele = new Television(); // instantiate the class Television
```

Comment:
Ook hier is de naame van de variabele niet heel goed gekozen. Je kosrt "television" af naar "tele", maar dat maakt het minder leesbaar. Wanneer je project groter wordt, weet je niet meer wat een "tele" is als je de context er niet meer bij ziet.